hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemRandomRead.java

```diff
@@ -412,6 +413,17 @@ public void testSequentialReadAfterReverseSeekPerformance()
   }
 
   @Test
+  @Ignore(
+      "ABFS accounts are primarily for customers to use with HNS property "
```

Comment:
Code comment needs only the JIRA number.

And the JIRA is still missing description.